src/gcinfo/gcinfoencoder.cpp

```diff
@@ -1217,7 +1217,7 @@ void GcInfoEncoder::Build()
         // (after, of course, adding the size of the call instruction to get the return PC).
         callSite += m_pCallSiteSizes[callSiteIndex] - 1;
 
-        _ASSERTE(DENORMALIZE_CODE_OFFSET(NORMALIZE_CODE_OFFSET(callSite)) == callSite);
+        _ASSERTE(DENORMALIZE_CODE_OFFSET(NORMALIZE_CODE_OFFSET(callSite + 1)) == (callSite + 1));
```

Comment:
The fix should be likely in the JIT - the offset value passed from the ARM JIT should not have the low bit set.

These macros are used to compress the data. If the start of instruction is always 2 byte aligned, it is not needed to encode the lowest bit - because of it should be always zero.